week3/clicelee/todo/db.json

```
@@ -0,0 +1,34 @@
+{
+  "todos": [
+    {
+      "id": 1,
+      "title": "할일 1",
+      "createdAt": "Mon Sep 25 2023",
+      "completed": true
+    },
+    {
+      "id": 2,
+      "title": "할일2",
+      "createdAt": "Mon Sep 25 2023",
+      "completed": true
+    },
+    {
+      "id": 3,
+      "title": "할일3",
+      "createdAt": "Mon Sep 25 2023",
+      "completed": true
+    },
+    {
+      "id": 4,
+      "title": "할일4",
+      "createdAt": "Mon Sep 25 2023",
+      "completed": false
+    },
+    {
+      "id": 5,
+      "title": "할일5",
+      "createdAt": "Mon Sep 25 2023",
+      "completed": false
+    },
+  ]
```

배열 끝부분에 쉼표(,)가 들어가서 db.json에서 문법 에러가 나서 안된 것 같습니다. 
끝부분에 쉼표만 제거해주면 수정도 잘 됩니다! 
```suggestion
    {
      "id": 5,
      "title": "할일5",
      "createdAt": "Mon Sep 25 2023",
      "completed": false
    }
  ]
```

---

다시보니 json-server 자체가 not found라고 뜨시는거였네요
글로벌로 (-g) 설치하신것 맞나요? 
제가 실행해보니 수정은 잘 됩니다!